src/common/options.cc

```diff
@@ -3617,19 +3643,19 @@ std::vector<Option> get_global_options() {
 
     Option("filestore_rocksdb_options", Option::TYPE_STR, Option::LEVEL_ADVANCED)
     .set_default("")
-    .set_description(""),
+    .set_description("Options to pass through when RocksDB is used as the KeyValueDB for filestore."),
 
     Option("filestore_omap_backend", Option::TYPE_STR, Option::LEVEL_ADVANCED)
+    .set_enum_allowed({"rocksdb", "leveldb"})
     .set_default("rocksdb")
-    .set_description(""),
-
+    .set_description("The KeyValueDB to use for filestore metadata (ie omap).  Choices are: rocksdb (preferred) and leveldb"),
```

Comment:
missing newline

Reply:
can leave off the 'choices'.  antyhing beyond 1 sentence should go in long_description, but i don't think it's needed here